build/common.sh

```diff
@@ -317,6 +317,10 @@ function kube::build::has_docker() {
   which docker &> /dev/null
 }
 
+function kube::build::has_ip() {
```

Comment:
This function name isn't descriptive (having an IP is different than having the ip tool), and lacks a comment similar to other functions in this script.

Reply:
`ip` is quite vague in the Linux world, but it is appropriate in this context based on the has_docker function

Reply:
Perhaps "has_iproute2" which is the package name?